mdal/mdal_driver_manager.cpp

```diff
@@ -38,6 +38,32 @@
 #include "frmts/mdal_xdmf.hpp"
 #endif
 
+std::string MDAL::DriverManager::getUris( const std::string &file ) const
+{
+  if ( !MDAL::fileExists( file ) )
+  {
+    MDAL::Log::error( MDAL_Status::Err_FileNotFound, "File " + file + " could not be found" );
+    return std::string();
+  }
+
+  for ( const auto &driver : mDrivers )
+  {
+    if ( ( driver->hasCapability( Capability::ReadMesh ) ) &&
+         driver->canReadMesh( file ) )
+    {
+      std::unique_ptr<Driver> drv( driver->create() );
+
+      if ( drv->load( file ) )
```

Comment:
we do not want to use load(file) here. canReadMesh() is enough... 

`canReadMesh` is fast function that should tell you exactly that file can be loaded. 
`load` is very slow function (sometimes) and you do not want to read 10GB of text file to throw it away second later :)